docs/rootchainFunctions.md

```diff
@@ -0,0 +1,60 @@
+# PLASMA MVP Rootchain Documentation
+## RootChain.sol
+**function** `RootChain()` </br>
+Contract constructor that sets the validator of the child chain.
+
+**function** `submitBlock(bytes32 root)` </br>
+The validator submits the merkle root of a child chain block. </br>
+`@param bytes32 root`: merkle root of the child chain
+
+**function** `deposit(uint blocknum, bytes txBytes)` </br>
+Sender can deposit Eth into the smart contract, which will become redeemable on the child chain. </br>
+`@param uint blocknum`: the current child chain block number </br>
+`@param bytes txBytes`: the transaction bytes of the deposit, which is an encoded list of 15 elements.
```

Comment:
"the transaction bytes of the deposit, which is an RLP-encoded list of 15 elements."